Base/usr/share/man/man1/rev.md

```diff
@@ -0,0 +1,27 @@
+## Name
+
+rev - reverse lines
+
+## Synopsis
+
+```*sh
+$ rev [files...]
+```
+
+## Description
+
+`rev` reads the specified files line by line, and prints them to standard
```

Comment:
Maybe something like `rev reads the specified files line by line and prints each line reversed to standard output`? When I read this the first time I thought it was going to print the lines in the right character order, but the file would be printed bottom to top.

Reply:
I clarified this in my latest commit.

Reply:
Awesome, thank you.